google-cloud-bigquery/lib/google/cloud/bigquery/model/metrics.rb

```diff
@@ -0,0 +1,702 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Cloud
+    module Bigquery
+      class Model
+        # Information about a single training query run for the model.
+        class TrainingRun
```

Comment:
Personally I would find it easier to keep things straight if this file were `training_run.rb`.

Reply:
Done.